src/pages/workspaces/workspace/analysis/modals/ComputeModal.js

```diff
@@ -1780,11 +1781,10 @@ export const ComputeModalBase = ({
     return div({ style: { ...computeStyles.whiteBoxContainer, marginTop: '1rem' } }, [
       h(IdContainer, [
         id => h(div, { style: { display: 'flex', flexDirection: 'column' } }, [
-          label({ htmlFor: id, style: computeStyles.label }, ['Persistent disk']),
-          div({ style: { marginTop: '0.5rem' } }, [
+          label({ htmlFor: id, style: computeStyles.label }, ['Persistent disk', div({ style: { marginTop: '0.5rem' } }, [
```

Comment:
Remove the `htmlFor` attribute here. I don't understand why this text about the persistent disk is being linked to something.